python/pyspark/sql/functions.py

```diff
@@ -1829,6 +1829,112 @@ def bitwise_not(col: "ColumnOrName") -> Column:
     return _invoke_function_over_columns("bitwise_not", col)
 
 
+@try_remote_functions
+def bit_count(col: "ColumnOrName") -> Column:
```

Comment:
Should also add them to *.rst files for Python API reference documentation.

Reply:
Sorry. I forgot it.